### PR TITLE
Installation

### PR DESCRIPTION
**Pull request by Zehen-249**
Please make a new branch named Installation in the upstream repository and merge this PR into it.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -9,8 +9,11 @@
 **/.idea/
 *~
 .vscode
-*.sqlite
```
Comment (AjinkyaDahale) [inline .gitignore:12]: Let's not remove this without confirming why it's there. The solution might be to make these files by ourselves from the sql.

File: .gitignore [hunk 2]
```diff
@@ -9,8 +9,11 @@
...
-**/*.sqlite
+
+# commented
```
Comment (AjinkyaDahale) [inline .gitignore]: A comment saying "commented" is redundant.

File: .gitignore [hunk 3]
```diff
@@ -9,8 +9,11 @@
...
+# *.sqlite
```
Comment (AjinkyaDahale) [inline .gitignore]: Don't leave it commented. Just delete the line. These lines can be recovered from the git history later if needed.

File: .gitignore [hunk 4]
```diff
@@ -9,8 +9,11 @@
...
+# **/*.sqlite
+
```
Comment (AjinkyaDahale) [inline .gitignore]: For now, we will ignore this commit until we decide how to best maintain the database. However, if we choose to keep this, 1) the file is called `.gitignore`; and 2) the message can just say "track/do not ignore `.sqlite` files".

File: src/osdag/Common.py [hunk 5]
```diff
@@ -483,9 +483,9 @@ def is_valid_custom(self):
 Load_type1 = 'Concentric Load'
 Load_type2 = 'Leg Load'
 Strut_load = list((Load_type1, Load_type2))
-IMG_STRUT_1 = "ResourceFiles/images/bA.png"
-IMG_STRUT_2 = "ResourceFiles/images/bBBA.png"
-IMG_STRUT_3 = "ResourceFiles/images/back_back_same_side_angles.png"
+IMG_STRUT_1 = str(files("osdag.data.ResourceFiles.images").joinpath("bA.png"))
```
Comment (AjinkyaDahale) [inline src/osdag/Common.py:486]: Can you confirm if the `str(...)` call is necessary? As far as I know this is supposed to allow "path-like objects".

**Review by AjinkyaDahale — CHANGES_REQUESTED** [review]
Apart from the comments, the first (merge) commit should not exist. Could you try rebasing?

**Comment by AjinkyaDahale** [comment]
> Please make a new branch named Installation in the upstream repository and merge this PR into it.

Indeed it is probably time to change the branch name. Is there any reason to call it "installation" though?

File: .gitignore (hunk 1 above)
Comment (Zehen-249) [reply .gitignore:12]: if the sqlite file is absent the application do not run.

File: .gitignore (hunk 2 above)
Comment (Zehen-249) [reply .gitignore]: got it

File: .gitignore (hunk 3 above)
Comment (Zehen-249) [reply .gitignore]: done

File: src/osdag/Common.py (hunk 5 above)
Comment (Zehen-249) [reply src/osdag/Common.py:486]: it has to be in string format. otherwise it is causing error

**Comment by Zehen-249** [comment]
> > Please make a new branch named Installation in the upstream repository and merge this PR into it.
> 
> Indeed it is probably time to change the branch name. Is there any reason to call it "installation" though?

> these were the issue i got while installing the osdag so i named it this way

File: .gitignore (hunk 1 above)
Comment (AjinkyaDahale) [reply .gitignore:12]: Yes it doesn't, but sgould we include it or build it during installation is the question.

File: src/osdag/Common.py (hunk 5 above)
Comment (AjinkyaDahale) [reply src/osdag/Common.py:486]: OK so pyqt isn't that well integrated maybe.

File: .gitignore (hunk 4 above)
Comment (AjinkyaDahale) [reply .gitignore]: @Zehen-249 I gave option to choose "track" or "do not ignore" 😂.

**Comment by AjinkyaDahale** [comment]
> > these were the issue i got while installing the osdag so i named it this way

Could you get the exact error received?

**Pull request closed without merging**